history-of-quakers/original/12-chapter-10.adoc

```diff
@@ -553,7 +608,8 @@ and delivering them into the charge of the sergeant at-arms attending this house
 Ordered, that the thanks of this house be given to the mayor and magistrates of Harwich,
 for their care in this business.
 
-WILLIAM JESSOP, Clerk of the Common House of Parliament.`'
+[.signed-section-signature]
+William Jessop, Clerk of the Common House of Parliament.
```

Comment:
Just in the original version, we're missing two closing `--` to end embeds. One of them is below this line.